components/dashboard/src/admin/admin-menu.ts

```diff
@@ -4,29 +4,39 @@
  * See License-AGPL.txt in the project root for license information.
  */
 
-export const adminMenu = [
-    {
-        title: "Users",
-        link: ["/admin/users", "/admin"],
-    },
-    {
-        title: "Workspaces",
-        link: ["/admin/workspaces"],
-    },
-    {
-        title: "Projects",
-        link: ["/admin/projects"],
-    },
-    {
-        title: "Teams",
-        link: ["/admin/teams"],
-    },
-    {
-        title: "License",
-        link: ["/admin/license"],
-    },
-    {
-        title: "Settings",
-        link: ["/admin/settings"],
-    },
-];
+export function getAdminMenu(isBlockedRepositoriesUIEnabled: boolean) {
+    return [
+        {
+            title: "Users",
+            link: ["/admin/users", "/admin"],
+        },
+        {
+            title: "Workspaces",
+            link: ["/admin/workspaces"],
+        },
+        ...(isBlockedRepositoriesUIEnabled
+            ? [
+                  {
+                      title: "Blocked Repositories",
+                      link: ["/admin/blocked-repositories"],
+                  },
+              ]
+            : []),
```

Comment:
**suggestion(non-blocking)**: What do you think of moving this further below, as this is more related to abuse control of the instance than associated with the workspace, project, or team management?

Later on, we could rename this section to _Abuse_, _Safety_, or something similar that could contain more abuse controls. 💭 

| BEFORE | AFTER |
|-|-|
| <img width="1440" alt="Screenshot 2022-07-12 at 1 38 26 PM" src="https://user-images.githubusercontent.com/120486/178473141-dcbc4fa8-8de2-47e5-a7b6-fa4ec81c7c3f.png"> | <img width="1440" alt="Screenshot 2022-07-12 at 1 45 16 PM" src="https://user-images.githubusercontent.com/120486/178473143-4ebf731a-43e2-49d6-9eea-341fb10564cb.png"> |

Reply:
Thanks 👍 I've moved the new entry down in the list as you suggested.